src/renderer/renderElement.ts

```diff
@@ -366,7 +371,9 @@ export function renderElementToSvg(
       }
       node.setAttribute(
         "transform",
-        `translate(${offsetX || 0} ${offsetY || 0})`,
+        `translate(${offsetX || 0} ${
+          offsetY || 0
+        }) rotate(${degree} ${cx} ${cy})`,
```

Comment:
AFAIK the `cx` `cy` args do nothing. `rotate()` takes only a single arg. https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate

Reply:
Mhm.. it seems necessary for SVG. What is this sorcery.

Reply:
Ok, now I'm seeing it. It's not a CSS prop, but an SVG attribute, so nvm.